sql/src/main/java/io/crate/metadata/doc/DocIndexMetaData.java

```diff
@@ -237,12 +240,32 @@ private RowGranularity granularity(ColumnIdent column) {
     private Reference newInfo(Integer position,
                               ColumnIdent column,
                               DataType type,
+                              @Nullable String formattedDefaultExpression,
                               ColumnPolicy columnPolicy,
                               Reference.IndexType indexType,
                               boolean nullable,
                               boolean columnStoreEnabled) {
+        Symbol defaultExpression = null;
+        if (formattedDefaultExpression != null) {
+            ExpressionAnalyzer expressionAnalyzer
+                = new ExpressionAnalyzer(functions,
```

Comment:
I think it might make sense to create this only once? Maybe already in the constructor?